src/styles/GameboardContainer.module.css

```diff
@@ -77,7 +77,7 @@
 .notEnoughBalanceMsg {
   font-weight: bold;
   margin: 1rem auto 0.5rem auto;
-  color: red !important;
+  color: #F9BA37 !important;
```

Comment:
Cuesta un poco leer el texto con ese color. ¿Nos pidieron que lo cambiáramos de rojo a este?

<img width="371" alt="image" src="https://github.com/KenrouIt/cheddar-ecosystem/assets/30414399/269d1c69-387b-459e-8e0e-f889781a580c">

Reply:
> Cuesta un poco leer el texto con ese color. ¿Nos pidieron que lo cambiáramos de rojo a este?
> 
> <img alt="image" width="371" src="https://private-user-images.githubusercontent.com/30414399/335419891-269d1c69-387b-459e-8e0e-f889781a580c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcyNjY3OTQsIm5iZiI6MTcxNzI2NjQ5NCwicGF0aCI6Ii8zMDQxNDM5OS8zMzU0MTk4OTEtMjY5ZDFjNjktMzg3Yi00NTllLThlMGUtZjg4OTc4MWE1ODBjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjAxVDE4MjgxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNmOWJmYmZhYTRiMmNkZTBhYWNkZGEwYmNhNDcxYTVjNDU2NmI0ZmY2ZjFjNTBkOGRjNjIwMmExYTZlZTkzNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SAmYhSNwq7zKm1NWzu0M-mbVrVu1yDCMZhRG75MiaeM">

La tarea dice lo siguiente:
"
We have a warning message that says "You must have 555 (cheddar icon) to play.".

-Change it to "Must have 555 🧀 and Verified Human to Play."
-Make it orange
".

Me permito anticiparte que los criterios estéticos del jefe suelen ser "distintos" por así decir.. Por eso entre parentesis comenté que sugería ponerle un fondo blanco pero no lo voy a hacer hasta que lo pida.

Reply:
Black border to letters added